doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -0,0 +1,1337 @@
+
+* :ref:`lib_download_client` library:
+
+  * Removed the deprecated ``download_client_connect`` function.
+
+* :ref:`lib_fota_download` library:
+
+  * Added:
+
+    * The function :c:func:`fota_download_b1_file_parse` to parse a bootloader update file path.
+    * Experimental support for performing FOTA updates using an external download client with the Kconfig option :kconfig:option:`CONFIG_FOTA_DOWNLOAD_EXTERNAL_DL` and functions :c:func:`fota_download_external_start` and :c:func:`fota_download_external_evt_handle`.
+
+Security libraries
+------------------
+
+* :ref:`trusted_storage_readme` library:
+
+  * Added the Kconfig option :kconfig:option:`CONFIG_TRUSTED_STORAGE_STORAGE_BACKEND_CUSTOM` that enables use of custom storage backend.
+
+SUIT DFU
+--------
+
+* Added:
+
+  * Experimental support for :ref:`SUIT DFU <ug_nrf54h20_suit_dfu>` for nRF54H20.
+    SUIT is now the only way to boot local domains of the nRF54H20 SoC.
+  * Experimental support for DFU from external flash for nRF54H20.
+  * Experimental support for recovery image for application and radio cores for nRF54H20.
+
+Other libraries
+---------------
+
+* Added:
+
+  * The :ref:`dult_readme` library with experimental maturity level.
+  * The :ref:`lib_uart_async_adapter` library.
+
+* :ref:`app_event_manager` library:
+
+  * Added the :kconfig:option:`CONFIG_APP_EVENT_MANAGER_REBOOT_ON_EVENT_ALLOC_FAIL` Kconfig option.
+    The option allows to select between system reboot or kernel panic on event allocation failure for default event allocator.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added:
+
+  * The :file:`thingy91x_dfu.py` script in the :file:`scripts/west_commands` folder.
+    The script adds the west commands ``west thingy91x-dfu`` and ``west thingy91x-reset`` for convenient use of the serial recovery functionality.
+  * Support for format version ``1`` of the :file:`dfu_application.zip` file to :ref:`nrf_desktop_config_channel_script` Python script.
+    From |NCS| v2.7.0 onwards, the script supports both format version ``0`` and ``1`` of the zip archive.
+    Older versions of the script do not support format version ``1`` of the zip archive.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+|no_changes_yet_note|
```

Comment:
Added skip for MCUboot downgrade prevention (both the software semantic version based and security counter based) for updating S0/S1 image (MCUBoot self update). Thanks to that MCUboots downgrade prevention  do not preventing updating S0/S1 image while using a swap algorithm. S0/S1 Image version management is kept by NSIB control anyway.
ref.: NCSDK-27994, NCSDK-27969